chat-components/src/main/java/com/pubnub/components/chat/network/mapper/NetworkMessageMapper.kt

```diff
@@ -18,9 +17,9 @@ class NetworkMessageMapper(private val mapper: MapperManager) : Mapper<NetworkMe
             id = messagePayload.id,
             text = messagePayload.text,
             contentType = messagePayload.contentType,
-            content = messagePayload.content as DBCustomContent?,
```

Comment:
@kazydek cast to `DBCustomContent` removed. Now if the developer wants to get his own object he needs to call `message.content.mapToObject(MyCustomType::class)`